core/audits/network-requests.js

```diff
@@ -61,6 +64,9 @@ class NetworkRequests extends Audit {
         ((record.frameId === mainFrameId) || undefined) :
         undefined;
 
+      const entity = classifiedEntities.entityByUrl.get(record.url);
+      const entityName = entity?.name || undefined;
```

Comment:
nit: `entity?.name` would be undefined so the `||` might not be needed. L86 could just be `entity: entity?.name,` as well.